solidity/contracts/Oracle.sol

```diff
@@ -27,15 +36,11 @@ contract Oracle is Ownable {
     LINK = LinkToken(_link);
   }
 
-  function onTokenTransfer(
-    address _sender,
-    uint256 _amount,
-    bytes _data
-  )
-    public
-    onlyLINK
+  function onTokenTransfer(address _sender, uint _wei, bytes _data)
```

Comment:
I can live with consolidating to one line, but I think being explicit that it is a uint256 is helpful when generating function selectors. It's the kind of thing I've torn my hair out on because it's so small, but I think consistency in labeling types will be helpful in the long run.